docs/usermanual/component_reference.html

```diff
@@ -4276,10 +4289,32 @@ <h3 id="JMS_Point-to-Point_parms1">
     temporary queues will be used for the communication between the requestor and the server.
     This is very different from the fixed reply queue. With temporary queues the sending thread will block until the reply message has been received.
     With <span class="code">Request Response</span> mode, you need to have a Server that listens to messages sent to Request Queue and sends replies to 
-    queue referenced by <span class="code">message.getJMSReplyTo()</span>.</dd>
-    
+    queue referenced by <span class="code">message.getJMSReplyTo()</span>.
+</dd>
+
+<dt>
+<span class="code">Read</span>
+</dt>
+<dd> will read a message from an outgoing queue which has no listeners attached. This can be convenient for testing purposes.
+     This method can be used if you need to handle queues without a binding file (in case the jmeter-jms-skip-jndi library is used),
+     which only works with the JMS Point-to-Point sampler.
+     In case binding files are used, one can also use the JMS Subscriber Sampler for reading from a queue.
+</dd>
+
+<dt>
+<span class="code">Browse</span>
+</dt>
+<dd> will determine the current queue depth without removing messages from the queue, returning the number of messages on the queue. 
```

Comment:
Would it be possible to provide screenshots as it makes documentation much clearer